Tests/scripts/utils/compare_staging_graph.py

```diff
@@ -0,0 +1,189 @@
+# function to compare two zip files
+
+import argparse
+import filecmp
+import json
+import os
+from pathlib import Path
+import shutil
+from zipfile import ZipFile
+
+import difflib
+from contextlib import redirect_stdout
+import dictdiffer
+
+from ruamel.yaml import YAML
+from slack_sdk import WebClient
+
+yaml = YAML()
+
+
+def sort_dict(dct: dict):
+    for k, v in dct.items():
+        if isinstance(v, dict):
+            sort_dict(v)
+        if isinstance(v, list):
+            try:
+                v.sort()
+            except TypeError:
+                v.sort(key=lambda item: item.get("name"))
```

Comment:
just making sure - do we always have a name field?

Reply:
The problem here that we are comparing dictionaries with a nested list of dictionaries, and it's unsorted. We can't put dictionaries in a set, but we can sort them by a field that they always should contain.